src/main/java/uk/gov/companieshouse/scanupondemand/orders/api/service/ScanUponDemandCostCalculatorService.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.companieshouse.scanupondemand.orders.api.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.scanupondemand.orders.api.config.CostsConfig;
+import uk.gov.companieshouse.scanupondemand.orders.api.model.ItemCostCalculation;
+import uk.gov.companieshouse.scanupondemand.orders.api.model.ItemCosts;
+
+import static java.util.Collections.singletonList;
+import static uk.gov.companieshouse.scanupondemand.orders.api.model.ProductType.SCAN_UPON_DEMAND;
+
+/**
+ * Simple service class implemented for consistency across the three Item API implementations.
+ */
+@Service
+public class ScanUponDemandCostCalculatorService {
+
+    private static final String DISCOUNT_APPLIED = "0";
+    private static final String CALCULATED_COST = "3";
+    private static final String POSTAGE_COST = "0";
```

Comment:
Should these costs also be in costs.yaml?

Reply:
Good question. My judgement was that these were much less likely to change than the charge for an item. I will ask about this. 👍